src/python/CRABClient/ClientUtilities.py

```diff
@@ -788,7 +788,7 @@ def checkStatusLoop(logger, server, api, taskname, targetstatus, cmdname):
     logger.debug("Ended %s process." % ("resubmission" if cmdname == "resubmit" else "submission"))
 
 
-def execute_command(command=None, logger=None, timeout=None, redirect=True):
+def execute_command(command=None, logger=None, timeout=None, redirect=True, live=False):
```

Comment:
what does live do "in English" ?
can you help me remember what "redirect" does ?
where did you plan to use "live" ?

Reply:
Ah...my bad.
It should be `livelog`: print log as soon as it arrive from subcommand (I plan to use logger instead of print).
`redirect` is you want to capture subprocess stdout/stderr into python variable (available in `proc.communicate()` line below).

Reply:
after some testing we think that we do not need `live` option and can simply set `redirect=False`